BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Service/Form/Mixin/FormViewButtonsMixin.js

```diff
@@ -0,0 +1,70 @@
+import FlashMessageBag  from '../../../Service/FlashMessage/FlashMessageBag'
+import FlashMessage     from '../../../Service/FlashMessage/FlashMessage'
+import ApplicationError from '../../../Service/Error/ApplicationError'
+
+let FormViewButtonsMixin = (superclass) => class extends superclass {
+
+    /**
+     * @inheritDoc
+     */
+    preinitialize() {
+        super.preinitialize();
+        this.events = this.events || {};
+        this.events['click button.submit-continue-form'] = '_submit';
+        this.events['click button.delete-button'] = '_deleteElement';
+    }
+
+    /**
+     * @param {Object} event
+     * 
+     * @return {Object}
+     */
+    getStatusCodeForm(event) {
+        let statusCodeForm = {
+            '422': $.proxy(this.refreshRender, this),
+            '200': $.proxy(this.refreshRender, this),
+            '201': $.proxy(this._redirectEditElement, this)
+        };
+
+        if ($(event.currentTarget).hasClass('submit-continue-form')) {
+            statusCodeForm['201'] = $.proxy(this._redirectNewElement, this);
+        }
+
+        return statusCodeForm;
+    }
+
+    /**
+     * Redirect to new workflow profile view
+     *
+     * @param {mixed}  data
+     * @private
+     */
+    _redirectNewElement(data) {
+        let message = new FlashMessage(data, 'success');
+        FlashMessageBag.addMessageFlash(message);
```

Comment:
all controller don't add message in response on server, so this method don't work for all form